src/shared/stagedProgress.ts

```diff
@@ -40,6 +40,8 @@ export type Stage = {
 };
 
 export abstract class StagedProgress<T> {
+  // eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
```

Comment:
try bumping to v5 of eslint parser/plugins. They fixed this limitation

Reply:
Did not work, renaming the field w/o the `#`